as-modules/embedded/build.xml

```diff
@@ -63,7 +63,7 @@
             <maven-resource group="org.infinispan" artifact="infinispan-cachestore-jpa" />
         </module-def>
 
-        <module-def name="org.infinispan.cdi" slot="${infinispan.slot}">
+        <module-def name="org.infinispan.cdi.embedded" slot="${infinispan.slot}">
```

Comment:
We would need to mention the rename here: https://github.com/infinispan/infinispan/blob/master/documentation/src/main/asciidoc/upgrading/upgrading.adoc